runtime/common/src/crowdloan.rs

```diff
@@ -1631,6 +1673,27 @@ mod benchmarking {
 			);
 		}
 
+		poke {
+			let para_id = ParaId::from(1);
+			let cap = BalanceOf::<T>::max_value();
+			let first_period = 0u32.into();
+			let last_period = 3u32.into();
+			let end = T::Auctioneer::lease_period();
+
+			let caller: T::AccountId = whitelisted_caller();
+			let verifier: MultiSigner = account("verifier", 0, 0);
+			CurrencyOf::<T>::make_free_balance_be(&caller, BalanceOf::<T>::max_value());
+			Crowdloan::<T>::create(
+				RawOrigin::Signed(caller.clone()).into(),
+				para_id, cap, first_period, last_period, end, Some(verifier.clone()),
+			)?;
+			contribute_fund::<T>(&caller, para_id);
+			frame_system::Pallet::<T>::set_block_number(T::BlockNumber::max_value());
+		}: _(RawOrigin::Signed(caller.clone()), para_id)
+		verify {
+			assert_last_event::<T>(RawEvent::AddedToNewRaise(caller,para_id).into())
```

Comment:
did you check that this actually works?

```
cargo test -p polkadot-runtime-common --features runtime-benchmarks
```

I don't think you have done the right things to empty new raise here.

I think you might want to add a `NewRaise::remove()` here just to simulate if the `NewRaise` is empty.